shared/src/main/scala/drt/shared/Api.scala

```diff
@@ -205,6 +206,21 @@ object Arrival {
   }
 }
 
+sealed trait FeedSource
+
+case object ApiFeed extends FeedSource
```

Comment:
I feel like these should be name spaced somehow to show that they are flags - 
i.e. shared.AclFeed doesn't tell me that this is a flag, I would probably assume that this object IS the AclFeed representation in the app, rather than just a flag to indicate it's category.

Reply:
(either the namespace, or the name could indicate this)